src/templates/doc.js

```diff
@@ -180,24 +183,63 @@ const DocTemplate = ({ data, pageContext, path: pagePath }) => {
           />
         </SideNavigation>
         <MainContent>
-          <h1 className="balance-text">{frontmatter.title} <span className="font-weight-light ml-2 text-muted badge-light px-2 rounded text-smaller position-relative lh-1 top-minus-3">v{version}</span></h1>
+        <DropdownButton
+            className="float-right position-relative py-1"
+            size="sm"
+            variant="outline-info"
+            id="page-actions-button"
+            title={
```

Comment:
Can you explain why you had to do this? I would have thought the icon could just be a child element?